x/datadeal/client/cli/txDataVerificationVote.go

```diff
@@ -0,0 +1,56 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdDataVerificationVote() *cobra.Command {
```

Comment:
We usually name the tx cmd in the form of verbs. Also the name of transaction is `VoteDataVerification`.

```suggestion
func CmdVoteDataVerification() *cobra.Command {
```

Reply:
Ah, I agree about that. BTW, I just refer to the `CmdRegisterOracleVote()` in `x/oracle` module.

Reply:
I know. IMO, the name of that cmd should also be changed to `CmdVoteOracleRegistration`.

Reply:
I'll change the `CmdRegisterOracleVote` to `CmdVoteOracleRegistration` in this branch.